Alias information in Warehouse Schema documentation (#563)
rudderlabs/rudderstack-docs: data-warehouse-integrations/warehouse-schemas.mdx

| `context_<prop>`     | String, Integer  | `context_app_version: 1.2.3`, `context_screen_density: 2`      | -                                                                          |
| `uuid_ts`            | Timestamp        | `2020-04-26 07:31:54:735`                                      | Added by RudderStack for debugging purposes. Can be ignored for analytics. |

## Alias

For every [`alias`](https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/page/)/[`screen`](https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/screen/) call, RudderStack creates a record in the corresponding `aliases` table.

A sample `alias` call is shown below:

```javascript
rudderanalytics.alias("9bb5d4c2", "e6ab2c5e")
```

### Table: `aliases`

| Column               | Type             | Example value                                                  | Note                                                                        |
| :------------------- | :--------------- | :------------------------------------------------------------- | :------------------------------------------------------------------------- |

> | `user_id`                 | String           | `9bb5d4c2`                         | The group ID associated with the current user.                              |

Author: ```suggestion
| `user_id`                 | String           | `9bb5d4c2`                         | The new ID associated with the user.                              |
```

> | `anonymous_id`       | String           | `e6ab2c5e`                         | -                                                                          |

Author: ```suggestion
| `anonymous_id`       | String           | `e6ab2c5e`                         | The previous ID associated with the user.                                                                         |
```

> | `anonymous_id`       | String           | `e6ab2c5e`                         | The previous ID associated with the user.                                                                         |

Author: ```suggestion
| `previous_id`       | String           | `e6ab2c5e`                         | The previous ID associated with the user.                                                                         |
```
